src/ending/ceremony_and_credits.c

```diff
@@ -205,30 +205,30 @@ void reset_spline(void) {
     D_80287B20 = 0;
 }
 
-void func_80282434(UNUSED struct CinematicCamera *arg0) {
+void wrap_reset_spline(UNUSED CinematicCamera *arg0) {
```

Comment:
reset_spline_wrap? wrap as in 'this function is a wrapper for the other function'?

when wrap is in the beginning it means means different

Reply:
when it's before what does it mean for you ?

Reply:
'wrap reset spline' turns wrap into a verb. Which some might think that this means that the 'reset spline' gets wrapped. Rather than the intention which is to say that this is just a wrapper function.

A couple other ideas to name it. 'reset_spline_helper' 'reset_spline_wrapper' 'reset_spline_wrap' 'reset_spline_decorator'